src/main/java/com/algosenpai/app/chapters/ChapterSorting.java

```diff
@@ -0,0 +1,293 @@
+package com.algosenpai.app.chapters;
+
+import com.algosenpai.app.Question;
+import com.algosenpai.app.model.ReviewTracingListModel;
+import com.algosenpai.app.model.ReviewTracingModel;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+import java.util.HashSet;
+
+public class ChapterSorting {
+
+    private static Random random = new Random();
+    private static Scanner s = new Scanner(System.in);
+    /**
+     * Generates the question by using a random number to determine which of the
+     * sub-questions to ask.
+     * 
+     * @return A question class that contains the question and expected answer.
+     */
+
+    public static Question generateQuestions() {
```

Comment:
The javadoc comments should be directly above the related method/class without a space.